diffrax/_custom_types.py

```diff
@@ -84,6 +84,36 @@ class SpaceTimeTimeLevyArea(AbstractSpaceTimeTimeLevyArea):
     K: PyTree[Array]
 
 
+def resulting_levy_area(
+    levy_area1: type[AbstractBrownianIncrement],
+    levy_area2: type[AbstractBrownianIncrement],
+) -> type[AbstractBrownianIncrement]:
+    """A helper that returns the stricter Levy area.
+
+    **Arguments:**
+
+    - `levy_area1`: The first Levy area type.
+    - `levy_area2`: The second Levy area type.
+
+    **Returns:**
```

Comment:
Nit: this will need a new line here to render correctly in the docs.

Reply:
Oh yes, thanks.